modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -62,6 +62,19 @@ public class UpgradeCatchAllCheck {
 		_commerceAccountHelper.getCurrentCommerceAccount(null);
 	}
 
+	public void method(
+		HttpServletRequest httpServletRequest, String name) {
+		CookieUtil.get(httpServletRequest, "test123");
+		CookieUtil.get(httpServletRequest, name);
```

Comment:
@MicaelleSilvaa, isn't it the same case? Should we handle the 2 calls above really differently? Can you please clarify?

Honestly, I believe it seems to be a bug in UpgradeCatchAll, after all the 2 calls above have the same number of parameters with the exact same types

Reply:
In the method signature we declare the variable name and type, this is an error case when it is not possible to identify the type/name of the variable. We always use defaults: "null, test123 or 123".